AlphaWallet/Settings/ViewControllers/FeaturesTableViewController.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  FeaturesTableViewController.swift
+//  AlphaWallet
+//
+//  Created by Jerome Chan on 7/1/22.
+//
+
+import UIKit
+
+class FeaturesTableViewController: UITableViewController {
+
+    private static let cellIdentifier: String = String(describing: FeaturesTableViewController.self)
+
+    private var features: Features
+    private var keys: [FeaturesAvailable] = FeaturesAvailable.allCases
+
+    init(features: Features) {
+        self.features = features
+        self.keys = self.keys.sorted { beforeKey, afterKey in
+            beforeKey.rawValue < afterKey.rawValue
+        }
+        super.init(nibName: nil, bundle: nil)
+        tableView.dataSource = self
+        tableView.delegate = self
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    // MARK: - Table view data source
+
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        return 1
+    }
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return keys.count
+    }
+
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        var cell: UITableViewCell! = tableView.dequeueReusableCell(withIdentifier: FeaturesTableViewController.cellIdentifier)
+        if cell == nil {
+            cell = UITableViewCell(style: .subtitle, reuseIdentifier: FeaturesTableViewController.cellIdentifier)
```

Comment:
```swift
        if cell == nil {
            cell = UITableViewCell(style: .subtitle, reuseIdentifier: FeaturesTableViewController.cellIdentifier)
        }
```
How do I register the default UITableViewCell using the subtitle style?

Reply:
Oh, you can't. You'd have to do a dummy subclass. If you know it may return nil, then line 32 shouldn't be an implicit unwrap for clarity, even though it wouldn't crash.

Reply:
I checked the code and it seems to be creating new instances of UITableViewCell and not caching at all.
Switched to subclass.